docs/workflow/building/libraries/code-coverage.md

```diff
@@ -28,40 +28,30 @@ An issue need not be addressed in its entirety. We happily accept contributions
 
 You can perform code coverage runs for the entire repository locally by adding the `coverage` switch (assuming that  source and test assemblies are already built):
 
-    build -test -coverage
+    build libs -test -coverage
```

Comment:
the default libs subset doesn't build the tests right?

Reply:
Thanks :)